lib/commands/upload.js

```diff
@@ -48,6 +50,7 @@ function sendToGoogle(files, id) {
 
       return Promise.promisify(drive.files.update)(options)
         .catch(function(err) {
+          console.log(err);
```

Comment:
@hess-google do you mind tossing this on the same line as the fatal error. I like the inclusion here to make the message a bit more descriptive, but think the same line will make the current stack of error messages cleaner

```js
console.log('Google Drive returned a fatal error: '.red + err.message);
```